Markdown/MultiMarkdown.sublime-syntax

```diff
@@ -25,4 +26,12 @@ contexts:
     - match: ''
       push:
         - meta_scope: meta.content.multimarkdown
-        - include: scope:text.html.markdown
+        - match: ''
+          embed: scope:text.html.markdown
+          escape: $(?!$)  # should never match
+    # Replace with the following once
+    # https://github.com/sublimehq/sublime_text/issues/2395#issuecomment-662515815 is fixed.
+    # - match: ''
+    #   embed: scope:text.html.markdown
+    #   embed_scope: meta.content.multimarkdown
+    #   escape: $(?!$)  # should never match
```

Comment:
I _think_ this will result in the same scoping, without adding an Oniguruma overhead. It pushes an extra context down so when markdown does a `set` from `main` it will retain the `meta_scope`.

```suggestion
        - match: ''
          push:
            - include: scope:text.html.markdown
```

Reply:
That is correct, but I simply replaced it with a multi-push and an anonymous context. 🙃